src-electron/db/zap-schema.sql

```diff
@@ -257,6 +259,7 @@ CREATE TABLE IF NOT EXISTS "ATTRIBUTE" (
   "IS_SCENE_REQUIRED" integer,
   "IS_OPTIONAL" integer,
   "IS_REPORTABLE" integer,
+  "IS_NULLABLE" integer,
```

Comment:
And here, why integer?  Os is the boolean in the EVENT_FIELD table the wrong one?

Reply:
Booleans in the EVENT_FIELD are wrong. I'll change them to 'integer'. Why did they work in the first place, no idea.

Reading https://www.sqlite.org/datatype3.html, there is no separate boolean storage class. However sqlite is very fuzzy and supports a lot of data type aliases and such. I suspect 'boolean' magically works as an alias for `integer`.

Either way, I like to keep the raw storage classes, not the aliase in the schema, so I'll replace the booleans there into integers.

Reply:
Even though, now that I'm thinking about this some more.... Maybe it's more correct to go and fix all of the actual "boolean" fields to be 'boolean', and not 'integer', and trust the database? It's probably more portable against other DB engines to use the "data type" and not the "storage class" from sqlite.....